terraform-tests/postgres_test.go

```diff
@@ -435,4 +436,30 @@ var _ = Describe("postgres", Label("postgres-terraform"), Ordered, func() {
 		})
 	})
 
+	Context("auto_minor_version_upgrade", func() {
```

Comment:
I think we are missing testing the happy path when it is enabled and a major version is specified

Reply:
Great catch. Thank you very much